### DISCUSSION TITLE
Spendable/pending/pending total

### DISCUSSION THREAD

**Discussion by hajes** [Chia-Network/chia-blockchain-gui#1370] [Q&A]
### What happened?

I have started my wallet after a month, just to find out "0 spendable balance". All transaction confirmed, pending total balance is correct. Unable to send anything.

I have erased all wallet db data...synced again...everything back to normal.

I have created/received few offers. It used about 10% of balance...spendable balance almost 0 again. Right top corner "wallet green light/synced"

there is many errors in debug.log about double spend...looks like offers didn't go well.

### Version

1.6.1

### What platform are you using?

macOS

### What ui mode are you using?

GUI

### Relevant log output or stacktrace

```shell
wallet chia.wallet.wallet_node    : WARNING  SpendBundle has been rejected by the FullNode. {'error': 'DOUBLE_SPEND'....
```

**Comment by hajes** [comment]
erased wallet db...resynced...still 80% of my wallet not usable. Unsure, whether it is an issue with GUI or Chia.

there is a payment I didn't even do last night...I have allegedly moved 80% of my account. It looks like someone tried to steal content of my wallet by false offers?

**Comment by hajes** [comment]
uninstalled Chia GUI, removed all in `~/.chia`

installed again, synced...working again

**Comment by hajes** [comment]
there is really something wrong.

I have created two offers 3 XCH each in exchange for USDS, and they hog/reserve >50 XCH????

**Comment by MumfMeisterT** [comment]
You can send yourself a couple of coins and it will resolve this issue for you.

**Reply by hajes** [reply]
unable to send anything because whole balance is gone/pending ;-)

The only way to cancel is simply remove check from "sending few coins to yourself"...it cancels immediatelly.

Question is, why 2 x 3 XCH offers move >50 XCH to pending.

**Comment by esaung** [comment]
This is how coinset model (similar to UTXO) works on Chia. Each value of your coin can be any amount of XCH. So your 50XCH could exist as one coin, and when you make an offer that one coin is locked to prevent double spend issues, so although you are putting an offer for 3XCH, the whole coin is locked up.

As @MumfMeisterT mentions, one way to work around this is to send yourself 3XCH, and that will split the 50XCH coin into a 47 XCH coin and 3 XCH coin. So long as your offer is for 3XCH or less, then the 3XCH coin will get locked up. If you wanted to make 2 3 XCh coins for two different offers, you'll need to send yourself a value more then 3XCH (like 3.0000001) to prevent the wallet from just sending yourself back the existing 3XCH coin.

**Comment by hajes** [comment]
as already written - `it is hard to send anything if your whole balance is locked up`.

I do not see any logical reason why should 6 XCH offers lock whole balance. I understand 6XCH pending for offers. What is another 50+ XCH for? It makes no sense whatsoever. For me, it is a bug in network, rendering offers useless for anything serious.

**Comment by hajes** [comment]
What is the point of unwanted extra "offer" locking up your whole balance without owner's permission?

It is flaw in algo/code from business point of view, and utility of network.

**Comment by BrandtH22** [comment]
@hajes , as esaung explained, the locking up of the coins is intended to ensure you do not accidentally void an offer by spending the associated funds. Chia uses a coinset model like cash in your wallet. If you only have a $20 in your wallet and want to create an offer for a $1 gumball you have to use that entire $20 and wait to get $19 back in change. 

What happened above is that your xch coin has a value of 50xch so when you create an offer it has to lock that entire coin. To avoid this you can send a portion of the funds to yourself (like going to the bank and getting change for the $20) then create the offer with the smaller value coins (the chia logic will use only what it has to so if you have smaller coins to use in the offer it will). 

Since this ticket is voer a year old I am going to close it out but feel free to reopen if any additional questions arise.
